content/news/014.general_meeting_2024.en.md

```diff
@@ -0,0 +1,11 @@
+---
+published: 2024-12-05
+description: We reflect on an eventfull year
+title: DD-IX Generalassembly 2024
```

Comment:
DD-IX General Assembly 2024

//we should be consistent with wording, either "general assembly" or "general meeting", see also text below.

Reply:
General Meeting it is